content/productivity/crawl-planets/plugin.js

```diff
@@ -3,233 +3,332 @@
+        stepper.style.height = '24px';
+
+        let percent = document.createElement('span');
+        percent.innerText = `${stepper.value}%`;
+        percent.style.float = 'right';
+
+        stepper.onchange = (evt) => {
+            percent.innerText = `${evt.target.value}%`;
+            try {
+                this.maxEnergyPercent = parseInt(evt.target.value, 10);
+            } catch (e) {
+                console.error('could not parse energy percent', e);
+            }
+        }
+
+        let minimumEnergyAllowedLabel = document.createElement('label');
+        minimumEnergyAllowedLabel.innerText = '% energy to fill after capture';
+        minimumEnergyAllowedLabel.style.display = 'block';
+
+        let minimumEnergyAllowedSelect = document.createElement('input');
+        minimumEnergyAllowedSelect.type = 'range';
+        minimumEnergyAllowedSelect.min = '0';
+        minimumEnergyAllowedSelect.max = '100';
+        minimumEnergyAllowedSelect.step = '1';
+        minimumEnergyAllowedSelect.value = `${this.minimumEnergyAllowed}`;
+        minimumEnergyAllowedSelect.style.width = '80%';
+        minimumEnergyAllowedSelect.style.height = '24px';
+
+        let percentminimumEnergyAllowed = document.createElement('span');
+        percentminimumEnergyAllowed.innerText = `${minimumEnergyAllowedSelect.value}%`;
+        percentminimumEnergyAllowed.style.float = 'right';
+
+        minimumEnergyAllowedSelect.onchange = (evt) => {
+            if (parseInt(evt.target.value, 10) === 0) percentminimumEnergyAllowed.innerText = `1 energy`;
+            else
+                percentminimumEnergyAllowed.innerText = `${evt.target.value}%`;
+            try {
+                this.minimumEnergyAllowed = parseInt(evt.target.value, 10);
+            } catch (e) {
+                console.error('could not parse minimum energy allowed percent', e);
+            }
+        }
+
+        let levelLabelTo = document.createElement('label');
+        levelLabelTo.innerText = 'Min. level to capture';
+        levelLabelTo.style.display = 'block';
+
+        let levelTo = document.createElement('select');
+        levelTo.style.background = 'rgb(8,8,8)';
+        levelTo.style.width = '100%';
+        levelTo.style.marginTop = '10px';
+        levelTo.style.marginBottom = '10px';
+        planetLevels.forEach(lvl => {
+            let opt = document.createElement('option');
+            opt.value = `${lvl}`;
+            opt.innerText = `Level ${lvl}`;
+            levelTo.appendChild(opt);
+        });
+        levelTo.value = `${this.minPlanetLevelTo}`;
+
+        levelTo.onchange = (evt) => {
+            try {
+                this.minPlanetLevelTo = parseInt(evt.target.value, 10);
+            } catch (e) {
+                console.error('could not parse planet level', e);
+            }
+        }
+
+        let levelLabelFromMin = document.createElement('label');
+        levelLabelFromMin.innerText = 'Min. level to send energy from';
```

Comment:
Maybe a line break above min level to send?

Reply:
OK, adding more space